### PR TITLE
feat(ON-1052): 💄 pagination design update for the new demo space tour

### PR DESCRIPTION
**Pull request by abenmassaoud**
This PR introduces the new demo space tour

## Pull request type

Jira Link: [ON-1052](https://storyblok.atlassian.net/browse/ON-1052)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe):

## How to test this PR

https://user-images.githubusercontent.com/16685155/228671813-5f00ccdd-70d3-47f0-a5c1-1224c34592c0.mov

ℹ️ Turn the volume on to hear the voiceover. Sorry for the quality of the video, I had to compress it to be able to upload it.

🎨  Figma design available [here](figma.com/file/2C0AQC1ILQyhkbwirFf4bI/%5BSBD-81%5D-Sign-up-process-v3?node-id=110-4582&t=a5kCAwqGU24j7li2-0)

ℹ️ For the sake of simplicity in the implementation. The position of the first hint is not exactly set as in the Figma design. It has been moved from the "Home" story checkbox, to the top of the  Stories List page.

## What is the new behavior?

- A new "demo space" tour is being introduced by this PR
- The tour is made up of a modal and some hints applied on the Stories List page & the Visual Editor page.

## Other information

⚠️ This PR is linked to [another one in the storyfront repo](https://github.com/storyblok/storyfront/pull/3536). Please review it as well 🙏 

## For the reviewers

[Pull Request Checklist](https://www.notion.so/storyblok/Pull-Request-Checklist-caef66feee2f4ae3b898534c1e4c1275)

[ON-1052]: https://storyblok.atlassian.net/browse/ON-1052?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Pagination/pagination.scss [hunk 1]
```diff
@@ -47,21 +47,39 @@
...
     }
 
     &--disabled {
       color: $sb-dark-blue-25;
       cursor: not-allowed;
     }
 
-    &:hover {
-      background-color: $light-50;
+    &--withLabel {
```
Comment (lisilinhart) [inline src/components/Pagination/pagination.scss:68]: this is not a proper css class, it should be `with-label`

Comment (abenmassaoud) [reply src/components/Pagination/pagination.scss]: Good catch! Thanks @lisilinhart 
 It's an old habit I am struggling to get rid of 😄 We had an internal _(BEM like)_ naming convention in my previous job.
 
 I pushed the change ✅

**Comment by abenmassaoud** [comment]
@lisilinhart I don't know if this new pagination design will become the new standard or just a variation of it.
Maybe I can add a new variation to the pagination story for the storybook.

Let me know 🙂

**Review by lisilinhart — COMMENTED** [review]
Hi @adelbenmassaoud, i updated the storybook, you can run in the ds folder with `yarn dev`, but the new options, they switch the mode of the pagination, maybe it should only be available inside the `compact` or `carousel`?

Check here: https://storyblok-design-system-git-subtask-on-1052-storyblok-com.vercel.app/?path=/story/navigation-sbpagination--default

**Review by SiloGecho97 — APPROVED** [review]

**Review by fariacc — APPROVED** [review]

**Pull request merged**